config_example/phoenix.php

```diff
@@ -13,6 +13,7 @@
             'password' => '123',
             'db_name' => 'phoenix',
             'charset' => 'utf8', // optional
+            'connection' => null, // existing PDO instance that you need to be reused; use this or other parameters
```

Comment:
we can create another array block under mysql / pgsql e.g. custom_connection with only adapter and connection keys

Reply:
Good idea! 
I have just implemented that. I'm not sure if how I did is good enough... `null` as separate environment from now doesn't work so I came with a solution with anonymous class.